lib/task_helpers.rb

```diff
@@ -175,7 +175,7 @@ def self.validate_index(alias_name)
         fail "Alias '#{alias_name}' does not exist."
       end
 
-      actual_mapping = Elasticsearch::Model.client.indices.get_mapping(index: alias_name).values[0]
+      actual_mapping = Elasticsearch::Model.client.indices.get_mapping(index: alias_name).values[0]['mappings']
```

Comment:
@dianakhuang: FYI: I'm not sure if you are validating mappings, but with the Ruby lib I get different formats back for ES 0.9 and ES 1.5.  Could just be a Ruby lib issue.